core_backend/migrations/versions/2024_08_01_465368ca2bac_rename_tables_and_add_user_id_column.py

```diff
@@ -0,0 +1,223 @@
+    )
+    op.execute('ALTER INDEX "urgency-query_pkey" RENAME TO urgency_query_pkey')
+
+    op.rename_table("urgency-response", "urgency_response")
+    op.execute(
+        'ALTER INDEX "ix_urgency-response_urgency_response_id" '
+        "RENAME TO ix_urgency_response_urgency_response_id"
+    )
+    op.execute('ALTER INDEX "urgency-response_pkey" RENAME TO urgency_response_pkey')
+
+    op.rename_table("content-tag", "content_tag")
+
+    # Add user_id column to query_response
+    op.add_column("query_response", sa.Column("user_id", sa.Integer(), nullable=True))
+    op.create_foreign_key(
+        constraint_name="fk_query_response_user_id_user",
+        source_table="query_response",
+        referent_table="user",
+        local_cols=["user_id"],
+        remote_cols=["user_id"],
+    )
+    op.execute(
+        """
+        UPDATE query_response qr
+        SET user_id = q.user_id
+        FROM query q
+        WHERE qr.query_id = q.query_id
+        """
+    )
+    op.alter_column("query_response", "user_id", nullable=False)
+
+    # Add user_id column to query_response_feedback
+    op.add_column(
+        "query_response_feedback", sa.Column("user_id", sa.Integer(), nullable=True)
+    )
+    op.create_foreign_key(
+        constraint_name="fk_query_response_feedback_user_id_user",
+        source_table="query_response_feedback",
+        referent_table="user",
+        local_cols=["user_id"],
+        remote_cols=["user_id"],
+    )
+    op.execute(
+        """
+        UPDATE query_response_feedback qrf
+        SET user_id = q.user_id
+        FROM query q
+        WHERE qrf.query_id = q.query_id
+        """
+    )
+    op.alter_column("query_response_feedback", "user_id", nullable=False)
+
+    # Add user_id column to content_feedback
+    op.add_column("content_feedback", sa.Column("user_id", sa.Integer(), nullable=True))
+    op.create_foreign_key(
+        constraint_name="fk_content_feedback_user_id_user",
+        source_table="content_feedback",
+        referent_table="user",
+        local_cols=["user_id"],
+        remote_cols=["user_id"],
+    )
+    op.execute(
+        """
+        UPDATE content_feedback cf
+        SET user_id = q.user_id
+        FROM query q
+        WHERE cf.query_id = q.query_id
+        """
+    )
+    op.alter_column("content_feedback", "user_id", nullable=False)
+
+    # Add user_id column to urgency_rule
+    # op.add_column("content_tag", sa.Column("user_id", sa.Integer(), nullable=True))
+    # op.create_foreign_key(
+    #     constraint_name="fk_content_tag_user_id_user",
+    #     source_table="content_tag",
+    #     referent_table="user",
+    #     local_cols=["user_id"],
+    #     remote_cols=["user_id"],
+    # )
+    # op.execute(
+    #     """
+    #     UPDATE content_tag ct
+    #     SET user_id = c.user_id
+    #     FROM content c
+    #     WHERE ct.content_id = c.content_id
+    #     """
+    # )
+    # op.alter_column("content_tag", "user_id", nullable=False)
```

Comment:
To do: find out how content_tag table is used and decide whether to include user_id column or not.